src/utils/utils.py

```diff
@@ -21,7 +21,6 @@
 from torch.nn import init
 
 from allennlp.common.checks import ConfigurationError
-from allennlp.nn.util import last_dim_softmax
```

Comment:
Is this used anywhere?

Reply:
It is but there's also a definition in the utils.py file (line 659)- actually, it seems like the definition in the utils.py file is last_dim_softmax in allennlp's version. Why was this definition added in?
We'll have to deprecate this line anyways with the new AllenNLP update.

Reply:
I think because I was ripping something out of AllenNLP early on before I wanted to invest in their eco-system...should be safe to pick one of using theirs or this clone (probably the former).